components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/ApiFileDAOImpl.java

```
@@ -422,6 +422,38 @@ public Comment getCommentByUUID(String commentId, String apiId) throws APIMgtDAO
     }
 
     /**
+     * @see ApiDAO#addComment(Comment , String)
+     */
+    @Override
+    public void addComment(Comment comment, String apiId) throws APIMgtDAOException {
+
```

Better to throw unsupported operation to avoid any conflict.